docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -64,58 +37,87 @@ To apply the policies on a specific team use the `--policies-team` flag:
+* macOS 13 Ventura - 96 of 104
+* Windows 10 Enterprise - TODO
 
-# Apply the downloaded policies to Fleet.
-fleetctl apply -f cis-policy-queries.yml
-```
+For a list of specific checks which are not covered by Fleet, please visit the section devoted to each benchmark.
+
+### Audit vs. remediation
+Each benchmark has two elements:
+1. Audit - how to find out whether the host is in compliance with the benchmark
+2. Remediation - if the host is out of compliance with the benchmark, how to fix it
+
+Since Fleetd is currently read-only without the ability to execute actions on the host, Fleet does not implement the remediation portions of CIS benchmarks.
+
+To implement automated remediation, you can install a separate agent such as Munki, Chef, Puppet, etc. which has write functionality.
+
+### Manual vs. automated
+
+For both the audit and remediation elements of a CIS Benchmark, there are two types:
+1. Automated - the element can be audited or remediated without human intervention
+2. Manual - the element requires human intervention to be audited or remediated
+
+Fleet only implements automated audit checks. Manual checks require administrators to implement other processes to conduct the check.
+
+### MDM required
+Some of the policies created by Fleet use the [managed_policies](www.fleetdm.com/tables/managed_policies). This checks whether an MDM solution has turned on the setting to enforce the policy.
+Using MDM is the recommended way to manage and enforce CIS benchmarks. To learn how to set up MDM in Fleet, visit [here](/docs/using-fleet/mdm-setup).
+
+## Levels 1 and 2
+CIS designates various benchmarks as Level 1 or Level 2 to describe the level of thoroughness and burden that each benchmark represents.
+
+### Level 1
+
+Items in this profile intend to:
+- be practical and prudent;
+- provide a clear security benefit; and
+- not inhibit the utility of the technology beyond acceptable means.
+
+### Level 2
+
+This profile extends the "Level 1" profile. Items in this profile exhibit one or more of the following characteristics:
+- are intended for environments or use cases where security is paramount or acts as defense in depth measure
+- may negatively inhibit the utility or performance of the technology.
+
+## macOS 13.0 Ventura benchmark
+
+Fleet's policies have been written against v1.0 of the benchmark. Please refer to the "CIS Apple macOS 13.0 Ventura Benchmark v1.0.0 - 11-14-2022" PDF from the CIS website for full details.
+
+### Checks that require customer decision
+
+CIS has left the parameters of the following checks up to the benchmark implementer. CIS recommends that an organization make a conscious decision for these benchmarks, but does not make a specific recommendation.
+
+Fleet has provided both an "enabled" and "disabled" version of these benchmarks. When both policies are added, at least one will fail. Once your organization has made a decision, you can delete one or the other policy query.
```

Comment:
Just something to solve in the future:

This approach (having a enable and disable policies and the user having to delete them) doesn't work well with new versions of the CIS policies.

Scenario:

1. Customer applies policies to their Fleet, via `fleetctl apply -f ./ee/cis/macos-13/cis-policy-queries.yml`.
1. Customer manually removes the ones that they do not want/need via the UI.
1. Fleet releases a new version of `./ee/cis/macos-13/cis-policy-queries.yml` (with some bug fixes on the queries).
1. Customer applies policies to their Fleet to get the updates, via`fleetctl apply -f ./ee/cis/macos-13/cis-policy-queries.yml`.
1. Now customer has to do step (2) again because the previous step added the policies again.